client/my-sites/stats/checklist-banner/index.jsx

```diff
@@ -202,21 +198,14 @@ export class ChecklistBanner extends Component {
 }
 
 const mapStateToProps = ( state, { siteId } ) => {
-	const tasksFromServer = get( getSiteChecklist( state, siteId ), [ 'tasks' ] );
-	const tasks = tasksFromServer
-		? mergeObjectIntoArrayById( onboardingTasks, tasksFromServer )
```

Comment:
Sorry for this but we can't just delete the line of code since the response from the server-side can return extra information besides whether a task completes or not. Let me give you an example. The URL of the very first post can be changed based on themes. So the server-side finds and returns its URL then this `mergeObjectIntoArrayById` applies the new URL to the corresponding task. Here's the server-side code.
https://opengrok.a8c.com/source/xref/trunk/wp-content/lib/onboarding-checklist/blog.php#39

You don't need to do anything. I will handle this problem in my PR: https://github.com/Automattic/wp-calypso/pull/26721